cocos/2d/CCNode.h

```diff
@@ -168,14 +169,26 @@ class CC_DLL Node : public Ref
      */
     CC_DEPRECATED_ATTRIBUTE virtual void _setLocalZOrder(int z);
 
+    /** !!! ONLY FOR INTERNAL USE
+    * Sets the arrival order when this node has a same ZOrder with other children.
+    *
+    * A node which called addChild subsequently will take a larger arrival order,
+    * If two children have the same Z order, the child with larger arrival order will be drawn later.
+    *
+    * @warning This method is used internally for localZOrder sorting, don't change this manually
+    *
+    * @param orderOfArrival   The arrival order.
+    */
+    CC_DEPRECATED_ATTRIBUTE void _updateOrderOfArrival(void);
```

Comment:
yep, but please 

* remove `CC_DEPRECATED_ATTRIBUTE ` since it is not deprecated, it is used
* _updateOrderOfArrival -> updateOrderOfArrival
* remove `void`